### DISCUSSION TITLE
Not permitted to join the room, despite of having required room access permissions

### DISCUSSION THREAD

**Discussion by asifs0862** [mozilla/hubs#4185] [Hubs Cloud Questions]
Hi,

One of our user is not able to access the below room, where as myself, and my colleagues are able to access the same room with same room permission settings.

https://tlishub.kbr.com/RNDyoKX/?hub_invite_id=nG4vNgK/customer_experience_center/

Affected user is getting following message while trying to access the room.

**You are not permitted to join this room. Please request permission from the room creator.**

**Comment by brianpeiris** [comment]
@asifs0862 Hmm, if the app generated the URL for you like that, it seems to be a bug. It's seems incorrect that there's a part after the `hub_invite_id`. I suspect the URL will work if you remove that part, like so: https://tlishub.kbr.com/RNDyoKX/?hub_invite_id=nG4vNgK

**Reply by asifs0862** [reply]
We have generated this url with invite id, so that is not a problem. With this url, we all are able to access the room, except 1 particular user. In security setting, we have mentioned that user need account created to access the room with this url. User account is created for all users including the affected user as well, still this user is not able to access the room. But rest all are able to access.
